### ISSUE TITLE
LD friendly term response

### ISSUE THREAD

**Issue by carueda** [mmisw/orr-ont#61]
(summarized from recent email thread. Thanks Nick Car, @dr-shothair, @lewismc .  )

Currently, **term requests** are internally handled via SPARQL query.

By default (and this may actually vary depending on the default accept header -or lack thereof- from the requesting client), the response is in one of the SPARQL Response formats, like https://www.w3.org/TR/rdf-sparql-XMLres/.

Two desiderata here:

- LD Friendly response format.  We would like to determine and implement a better, Linked Data friendly format as the **default** behavior.  In other words, do serve such SPARQL query result format **only** when explicitly requested from the user or client application. One option is to serve the same (or very similar) format used to register the term's containing ontology. (In the case of SWEET, this would be "Turtle").

- Better contents of the term response itself, in particular regarding any blank nodes. Related issue #57 .

Note, the focus here is on the "semantic format" for term responses, whereas #57's, although closely related, is HTML presentation.

**Comment by nicholascar** [comment]
HTTP Content Negotiation calling for a SPARQL response for URI dereferencing (e.g. Accept: application/sparql-results-xml) cannot be the entire solution since the query used to generate the data returned is not known. One assumes DESCRIBE <URI> is used but this may not be the case or an accepted convention. Also a DESCRIBE query will not expand blank nodes related to the dereferenced URI via an object property predicate.

**Comment by dr-shorthair** [comment]
> Also a DESCRIBE query will not expand blank nodes related to the dereferenced URI via an object property predicate.

I don't think this is correct. DESCRIBE response is the prerogative of the service provider. CBD https://www.w3.org/Submission/CBD/  is one option that does expand blank nodes.
